### ISSUE TITLE
Write Unit Testing code for various functions

### ISSUE THREAD

**Issue by MrigankaGhosh** [MrigankaGhosh/Tokyo-Gul#1]
# JUnit Testing
Your job is to write JUnit tests for as many functions as possible.

**Comment by NusaibaNizam** [comment]
Hello @MrigankaGhosh , May I work o this?

**Comment by MrigankaGhosh** [comment]
Sure @NusaibaNizam you can go ahead

**Comment by NusaibaNizam** [comment]
Sorry for the delay, I have set up the environment and will start working on it from tomorrow. May I use mockito for testing the CRUD operations?

**Comment by NusaibaNizam** [comment]
Sorry for the delay, I will be using mockito and JUnit5 for testing if it is okay.

**Comment by MrigankaGhosh** [comment]
@NusaibaNizam Yes, it's okay. Also, I'd appreciate it if you could start working asap.

**Comment by NusaibaNizam** [comment]
Yes, I am working on it. I have completed writing unit tests for UserServiceImpl. Should I create a PR request for that class should I create the PR request after I have written all the tests?

**Comment by MrigankaGhosh** [comment]
I think it would be better if you write all the test cases and then send a PR.

**Comment by NusaibaNizam** [comment]
@MrigankaGhosh while writing tests for BillingServiceImpl, I encountered an issue and suggested a fix in this [PR](https://github.com/MrigankaGhosh/Tokyo-Gul/pull/3) please review this, so that I can continue with the unit test for this class.

**Comment by MrigankaGhosh** [comment]
@NusaibaNizam can you send a PR to merge some of the Unit Tests?

**Comment by NusaibaNizam** [comment]
On it.

**Comment by NusaibaNizam** [comment]
Please do check this [issue](https://github.com/MrigankaGhosh/Tokyo-Gul/issues/4)

**Comment by NusaibaNizam** [comment]
This is a [PR](https://github.com/MrigankaGhosh/Tokyo-Gul/pull/6) for unit tests for service package
